build_tools/github_actions/runner/config/runner.env

```diff
@@ -1,3 +1,4 @@
 LANG=C.UTF-8
+RUNNER_GROUP=presubmit
```

Comment:
Should this be hardcoded to presubmit? Is this also used for postsubmit runners?

Reply:
Yes, when this was used in the postsubmit runner, it was modified during the install.  I saw some notes in the install doc (internally) that instruct to change the configuration to match the runner type.

I think these scripts were written for presubmit runners but they adapted to for postsubmit.  But the scripts didn't get updated and they probably should be.